submissions/isaiahUsiholo/database/transportQuery.sql

```diff
@@ -0,0 +1,34 @@
+SELECT * FROM registered_passengers;
+
+
+-- the total number of people who survived = 290;
```

Comment:
Your query is wrong, maybe you imported less than 891 people. Also select * from..... should be together with this new query